src/jit/emit.cpp

```diff
@@ -2969,9 +2999,10 @@ emitter::instrDesc  * emitter::emitNewInstrCallInd(int        argCnt,
                                                    VARSET_VALARG_TP GCvars,
                                                    regMaskTP  gcrefRegs,
                                                    regMaskTP  byrefRegs,
-                                                   emitAttr   retSizeIn)
+                                                   emitAttr   retSizeIn
+                                                   FEATURE_MULTIREG_RET_ONLY_ARG(emitAttr secondRetSizeIn))
 {
-    emitAttr  retSize = retSizeIn ? EA_ATTR(retSizeIn) : EA_PTRSIZE;
+    emitAttr       retSize = (retSizeIn != EA_UNKNOWN) ? retSizeIn : EA_PTRSIZE;
 
     bool gcRefRegsInScratch = ((gcrefRegs & RBM_CALLEE_TRASH) != 0);
```

Comment:
I don't think you understood my point. RAX and RDX are in RBM_CALLEE_TRASH, so at line 2976 gcRefRegsInScratch will be true if they hold gc refs. So I don't think we need the extra check at line 3022.